test/e2e/tls_security_profile_test.go

```diff
@@ -37,97 +37,39 @@ func atLeastVersionTLS12(v string) string {
 	return v
 }
 
-func TestTLSSecurityProfileConfiguration(t *testing.T) {
-	t.Skip("Changing apiserverConfig.Spec.TLSSecurityProfile now makes MCO rollout nodes which is disruptive for other tests. See https://issues.redhat.com/browse/MON-3959")
-	testCases := []struct {
-		name                  string
-		profile               *configv1.TLSSecurityProfile
-		expectedCipherSuite   []string
-		expectedMinTLSVersion string
-	}{
-		{
-			name:                  "no profile",
-			profile:               nil,
-			expectedCipherSuite:   manifests.APIServerDefaultTLSCiphers,
-			expectedMinTLSVersion: "VersionTLS12",
-		},
-		{
-			name: "old profile",
-			profile: &configv1.TLSSecurityProfile{
-				Type: configv1.TLSProfileOldType,
-				Old:  &configv1.OldTLSProfile{},
-			},
-			expectedCipherSuite:   configv1.TLSProfiles[configv1.TLSProfileOldType].Ciphers,
-			expectedMinTLSVersion: "VersionTLS10",
-		},
-		{
-			name: "intermediate profile",
-			profile: &configv1.TLSSecurityProfile{
-				Type:         configv1.TLSProfileIntermediateType,
-				Intermediate: &configv1.IntermediateTLSProfile{},
-			},
-			expectedCipherSuite:   configv1.TLSProfiles[configv1.TLSProfileIntermediateType].Ciphers,
-			expectedMinTLSVersion: "VersionTLS12",
-		},
```

Comment:
Not sure if upstream has tests for these two profiles, in which case it'd be good to configure the APIServer profile for them and keep these cases.

Reply:
the intermediate profile is still tested as the default one in `TestDefaultTLSSecurityProfileConfiguration`

Reply:
ACK, I saw they both use the same underlying API.